core/server/worker/src/main/java/alluxio/worker/page/PagedBlockStoreMeta.java

```diff
@@ -41,9 +44,11 @@
  * Metadata of the paged block store.
  */
 public class PagedBlockStoreMeta implements BlockStoreMeta {
-  public static final String DEFAULT_TIER = "DefaultTier";
-  public static final String DEFAULT_DIR = "DefaultDir";
-  public static final String DEFAULT_MEDIUM = "SSD";
+  // Paged block store does not support multiple tiers, so assume everything is in the top tier
+  public static final String DEFAULT_TIER =
+      Configuration.getString(PropertyKey.MASTER_TIERED_STORE_GLOBAL_LEVEL0_ALIAS);
+  // top tier is conventionally ramdisk
+  public static final String DEFAULT_MEDIUM = Constants.MEDIUM_MEM;
```

Comment:
Although I suspect in practice the page cache will not be in ramdisk, but for a paged block store in worker that needs to play nice with master, this convention still needs to be upheld, e.g. `alluxio.master.file.DefaultFileSystemMaster#getFileInMemoryPercentageInternal` checks  the in-memory percentage of a file by checking its blocks stored in the top tier.